test/transport_web/controllers/user_controller_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule TransportWeb.UserControllerTest do
+  use TransportWeb.ConnCase
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+  import Plug.Test
+
+  doctest TransportWeb.UserController
+
+  describe "GET /user/organizations" do
+    test "logged in", %{conn: conn} do
+      use_cassette "client/me-1" do
+        with path <- conn |> user_path(:organizations),
```

Comment:
The use of `with` statement is overkill here, you don't match arguments against a pattern.
I would prefer: 
```elixir
path = conn |> user_path(:organizations)
conn = conn |> init_test_session(current_user: %{"apikey" => "secret"}) |> get(path)
conn
|> html_response(200)
|> assert
```

Side effect: it less looks like brainfuck!